tinyengine/engine.py

```diff
@@ -1,18 +1,14 @@
+import importlib
+import pkgutil
+
 import pygame
-from pygame.locals import *
```

Comment:
There are some features that is imported by locals. Did you checked if the code is not using any of those?

Reply:
same